gamestore/src/main/java/com/company/gamestore/model/Fee.java

```diff
@@ -0,0 +1,75 @@
+package com.company.gamestore.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@Entity
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@Table(name = "fee")
+public class Fee implements Serializable {
+    // note backend product_type name is set to 50
+    @Id
+    @Column(name = "product_type")
+    @Size(max = 50, message = "product type must be 50 chars or less")
+    private String productType;
+
+    @Column(name = "fee")
+    @NotNull(message = "fee cannot be null")
+    @DecimalMin(value = "0.0", inclusive = false, message = "fee must be positive")
```

Comment:
edit this annotation to make it so that a user can't enter a decimal where the integer part is more than 8 digits and fraction part is more than 2 digits